src/sessions/session-manager.ts

```diff
@@ -66,38 +66,19 @@ export class SessionManager {
   }
 
   getPendingSession (params: ISessionIdAndConversationId): IPendingSession | undefined {
```

Comment:
should this type change?

Reply:
Are you referring to only having it as a conversationId param? I created [PCM-1767](https://inindca.atlassian.net/browse/PCM-1767) for that because changing it will be a little bit of a bigger lift than just getting LA == 1 to work.